simul.spec

```diff
@@ -0,0 +1,105 @@
+%global daos_major 0
+
```

Comment:
We're no longer doing this.  It can be removed.

```suggestion
```

Reply:
Line removed